docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "2"
+
+services:
+
+  github-api:
+    image: databraiddb/github-api:latest
+    ports:
+      - 8002:8002
```

Comment:
isn't it the case that we want to map to 8000 in the container? I noticed similar mapping in sheets-api

Reply:
@Gwillison415 See line 10

Reply:
@Gwillison415 and also [.env](https://github.com/databraid-dashboard/github-spa/blob/dev/.env)